mbed_app.json

```diff
@@ -27,6 +27,12 @@
         "AIN_4": "A4",
         "AIN_5": "A5",
         "AOUT":  "A5",
+        "AOUT_0": "A0",

```

Comment:
Are all this AOUT_x pins really analog out? I could not find this in any specifications. For instance, for  some targets like Freescale, they have just one DAC pin.